### PR TITLE
config: validate context-sections and show all available values

### PR DESCRIPTION
**Pull request by anthraxx**
When setting an illegal value, fall back to the default sections.

### CODE REVIEW

File: pwndbg/commands/context.py [hunk 1]
```diff
@@ -43,6 +43,19 @@ def clear_screen():
                                                   'which context sections are displayed by default (also controls order)')
 
 
+@pwndbg.config.Trigger([config_context_sections])
+def validate_context_sections():
+    valid_values = [context.__name__.replace('context_', '') for context in context_sections.values()]
+    for section in get_context_sections():
+        if section not in valid_values:
+            config_context_sections.value = config_context_sections.default
+            return
```
Comment (disconnect3d) [inline pwndbg/commands/context.py]: Can we print some hint/warn/error about `valid_values` and that we are switching back to default?

Comment (anthraxx) [reply pwndbg/commands/context.py]: it existed once... i swear... weird well the hell it vanished... :open_mouth:

**Comment by anthraxx** [comment]
adjusted adding warning and valid values, the setting of default values will be shown automatically below

**Comment by Kyle-Kyle** [comment]
After this change, what should I do if I don't want any context sections?

**Comment by disconnect3d** [comment]
@Kyle-Kyle haha, this is super good question :heart: will bring it back!

**Comment by disconnect3d** [comment]
@Kyle-Kyle will be there after merging https://github.com/pwndbg/pwndbg/pull/409

**Pull request merged**